examples/features/metadata_interceptor/client/main.go

```diff
@@ -16,7 +16,8 @@
  *
  */
 
-// Binary client is an example client.
+// Binary client demonstrates how to send and receive metadata in RPC headers
```

Comment:
This client only receives metadata. It does not send any.

Reply:
Done